src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -48,7 +49,13 @@ let nbCount: number = 0;
                 <ng-content></ng-content>
             </ng-template>
 
-            <button class="datagrid-column-title" *ngIf="sortable" (click)="sort()" type="button">
+            <button 
+              class="datagrid-column-title" 
+              [attr.aria-label]="commonStrings.sortColumn"
+              *ngIf="sortable" 
+              (click)="sort()" 
+              type="button">
+                <ng-container  *ngTemplateOutlet="columnTitle"></ng-container>
                 <ng-container *ngTemplateOutlet="columnTitle"></ng-container>
```

Comment:
Is it me or do you also see duplicates here 👀?

Reply:
Copy/Paste!